.gitlab-ci.yml

```diff
@@ -291,6 +291,7 @@ API:
   script:
     - schutzbot/deploy.sh
     - /usr/libexec/tests/osbuild-composer/api.sh ${TARGET}
+    - /usr/libexec/tests/osbuild-composer/api_v2.sh ${TARGET}
```

Comment:
We don't need to run this for each target. I know they're skipped in the shell script but this will result in a VM being provisioned and immediately torn down. Just add it for 1 target (see centos defined below).